controllers/spaceprovisionerconfig/spaceprovisionerconfig_controller.go

```diff
@@ -203,7 +204,8 @@ func determineMemoryUtilizationReadyState(spc *toolchainv1alpha1.SpaceProvisione
 
 	// the memory utilitzation is ok if it is below the threshold in all node types
 	for _, val := range spc.Status.ConsumedCapacity.MemoryUsagePercentPerNodeRole {
-		if uint(val) >= spc.Spec.CapacityThresholds.MaxMemoryUtilizationPercent {
+		//the value of this is not going beyond 100 and it won't overflow, hence its okay to ignore the overflow linter error
+		if uint(val) >= spc.Spec.CapacityThresholds.MaxMemoryUtilizationPercent { // nolint:gosec
```

Comment:
this is not true either - it's fine to cast the val to `uint` because we don't expect that it would be negative.
But it can be modified so you cast `spc.Spec.CapacityThresholds.MaxMemoryUtilizationPercent` to int instead